README.md

```diff
@@ -438,10 +438,10 @@ the proxy's internal port to the host. For example, you can use:
 ```shell
 docker run --publish <host-port>:<proxy-port> \
     gcr.io/cloud-sql-connectors/cloud-sql-proxy:2.12.0 \
```

Comment:
What if we changed this version to "latest" and removed the release please tags?

Reply:
Good idea, done!